spec/lib/workos/user_management_spec.rb

```diff
@@ -356,4 +356,37 @@
       end
     end
   end
+  describe '.authenticate_user_password' do
+    context 'with a valid password' do
+      it 'returns user' do
+        VCR.use_cassette('user_management/authenticate_user_password/valid') do
+          authentication_response = WorkOS::UserManagement.authenticate_user_password(
+            email: 'willman@blips.app',
```

Comment:
Let's leave real emails out of the public SDK test suite 😅 I've been manually editing the VCR cassettes as well to remove this.

Reply:
sorry about that!  Just removed the actual email from the VCR and test